.github/workflows/javadoc-publication.yaml

```
@@ -66,12 +66,29 @@ jobs:
         uses: SonarSource/jfrog-setup-wrapper@58546d2106f9d38c37590b7f125f85b0d1fa4cee # 3.5.0
         with:
           jfrogAccessToken: ${{ fromJSON(steps.secrets.outputs.vault).artifactory_access_token }}
+      - name: Create a path filter from the groupId for artifactory build search
+        id: filter
+        run: |
+          jfrog rt curl -X GET "/api/build/${{ github.event.repository.name }}/${{ steps.get_version.outputs.build }}" > build-info.json
+          groupId=$(jq -r '.buildInfo.modules[0].id' build-info.json | cut -d':' -f1)
+          echo "groupID: $groupId"
+          groupIdPath=$(echo "${groupId}" | tr '.' '/')
+          echo "path: $groupIdPath"
```

I would put the four lines in two lines:
```suggestion
          groupIdPath=$(jq -r '.buildInfo.modules[0].id' build-info.json | cut -d':' -f1 | tr '.' '/')
          echo "path: $groupIdPath"
```